node_exporter.go

```diff
@@ -110,6 +111,21 @@ func loadCollectors(list string) (map[string]collector.Collector, error) {
 		}
 		collectors[name] = c
 	}
+	for name, enabled := range collectorsEnabled {
```

Comment:
Indeed, I missed that when implementing. I'll look into that as soon as I get to it.

But first we need to decide if I should change the map-implementation for the flags above, because that would affect the implementation here, I suspect.